contracts/bprotocol/avatar/AvatarBase.sol

```diff
@@ -34,6 +34,11 @@ contract AvatarBase is Exponential {
     // Liquidation cToken
     ICToken public liquidationCToken;
 
+    modifier onlyAvatarOwner() {
```

Comment:
should we allow it for delegator? no, right?

Reply:
no